### PR TITLE
Improve BreadCrumb generation in Detail View.

### PR DESCRIPTION
**Pull request by thealmightyone**
ESI-10750

### CODE REVIEW

**Comment by thayashi** [comment]
breadcrumb

**Comment by thayashi** [comment]
Also, could you please put the JIRA Ticket number in the commit message?

**Comment by thealmightyone** [comment]
@thayashi Can you post what you had in the address bar at that time? I can't reproduce this behaviour.

**Comment by thayashi** [comment]
http://192.168.33.10:8055/#/v1.0/sites/1a719081-a9a2-405d-b768-a8f9d4360161/ese_devices/e795d275-8097-4315-a337-2878e6919766/ese_physical_ports/3b95da57-83d3-42c7-8124-65bf6b229ef0

File: app/js/views/detailView.js [hunk 1]
```diff
@@ -193,11 +194,13 @@ export default class DetailView extends View {
         result.push({
           title: ancestor.get('name'),
           url: modelFragment
-        },
-          {
+        });
+        if (ancestors.length === 1 || index !== 0) {
```
Comment (stepek) [inline app/js/views/detailView.js]: ```if (ancestors.length === index + 1) {``` should fix this issue.

Comment (stepek) [inline app/js/views/detailView.js]: screenshot-localhost 8080-2017-02-02-08-58-31

**Comment by thayashi** [comment]
LGTM

**Pull request merged**